tests/e2e/specs/admin-tracking.test.js

```diff
@@ -34,8 +34,11 @@ import {
 
 async function toggleOptIn() {
 	await page.waitForSelector( '#googlesitekit-opt-in' );
-	await expect( page ).toClick( '#googlesitekit-opt-in' );
-	await page.waitForResponse( ( res ) => res.url().match( 'wp/v2/users/me' ) );
+	await pageWait();
```

Comment:
Hmm, this feels hacky. Maybe good enough if it makes the test more stable, but why is timing even an aspect here, since below it should wait for these two things anyway?

Reply:
It is a bit hacky and I'm not particularly happy about it either but my guess is that there is some race condition that is happening when attempting to click the element as soon as it appears. Perhaps some odd implementation detail of how our `Checkbox` component uses [`MDCCheckbox`](https://github.com/material-components/material-components-web/tree/v2.3.0/packages/mdc-checkbox) internally.  This one was particularly difficult to debug as even running the same tests on travis with `DEBUG_*` variables set caused it to consistently pass with no other changes. Let's run the tests a few times to make sure this doesn't come up again - it wouldn't be the first case where adding a wait like this fixed a failing test.